packages/dds/merge-tree/src/client.ts

```diff
@@ -292,6 +292,20 @@ export class Client {
             accum, start, end, splitRange);
     }
 
+    public getGCData(handle: IFluidHandle, handleCollectingSerializer: IFluidSerializer): void {
```

Comment:
@DLehenbauer it wasn't immediately clear how to leverage this in matrix as it seems to use a side table for handles

Reply:
We just need to stringify all cells in the current set of rows/cols, right?
```ts
for (let r = 0; r < this.rowCount; r++) {
    for (let c = 0; c < this.colCount; c++) {
        handleCollectingSerializer.stringify(this.getCell(r, c), handle);
    }
}
```
(If it were less pressing, we could skip regions of empty cells by leveraging the tiled quadtree-like cell storage.)